packages/core/src/browser/style/sidepanel.css

```diff
@@ -56,19 +56,19 @@
 }
 
 .p-TabBar.theia-app-left .p-TabBar-tab {
-    border-left: var(--theia-panel-border-width) solid transparent;
```

Comment:
@nmorenor I'm not sure which IDE you're using, but can you revert all the unnecessary changes like this?

Reply:
Using vscode, I'm not sure why it changed the format on save, perhaps an extension. But those white space changes are already reverted.